apps/omg_db/lib/omg_db/rocks_db.ex

```diff
@@ -92,12 +92,10 @@ if Code.ensure_loaded?(:rocksdb) do
       GenServer.call(server_name, :competitors_info, @one_minute)
     end
 
-    @spec exit_info({pos_integer, non_neg_integer, non_neg_integer}, atom) :: {:ok, map} | {:error, atom}
```

Comment:
removing specs always looks suspicious - why?

Reply:
the rest doesn't have it and this one was incorrect, so I just went with the majority. The specs are in the behavior module.

I can apply them here as well, if there's consensus